hpack/hpack.py

```diff
@@ -362,15 +366,20 @@ def _encode_literal(self, name, value, indexing, huffman=False):
             name_len[0] |= 0x80
             value_len[0] |= 0x80
 
-        return b''.join([prefix, bytes(name_len), name, bytes(value_len), value])
+        return b''.join([indexbit, bytes(name_len), name, bytes(value_len), value])
 
-    def _encode_indexed_literal(self, index, value, huffman=False):
+    def _encode_indexed_literal(self, index, value, indexbit=INDEX_INCREMENTAL, huffman=False):
```

Comment:
With huffman we could probably skip the default, but let's not do it as part of this PR. Keep it on your to-do list, we'll keep each of these clean.